scenario/consistency_checker.py

```diff
@@ -371,6 +387,13 @@ def check_config_consistency(
                 f"but is of type {type(value)}.",
             )
 
+        elif expected_type_name in validators and not validators[expected_type_name](
+            value,
+        ):
```

Comment:
This line break is annoying (splitting the function call like that makes it hard to read). Might be a good use case for walrus?
```suggestion
        elif (validator := validators.get(expected_type_name)) and validator(value):
```

Reply:
I have a particular dislike for the walrus operator (:older_man:) but I agree the wrapping could be better (I lazily let `tox -e fmt` do it).

Reply:
Adjusted (I think @PietroPasotti does like the walrus, so he might prefer your suggestion).

Reply:
Sometimes I do like the :seal:, but I'm also one with the laziness of `tox -e fmt`. Can go both ways. Third option would be to make the line shorter by lifting 

`validator = validators.get(expected_type_name))`

out of the expression and into the outer scope.
I'd do that.

Reply:
> Third option would be to make the line shorter by lifting
> 
> `validator = validators.get(expected_type_name))`
> 
> out of the expression and into the outer scope. I'd do that.

Sounds good to me; I'll make that change.